internal/handlers/pg/msg_createindexes.go

```diff
@@ -16,24 +16,116 @@ package pg
 
 import (
 	"context"
+	"errors"
+	"fmt"
+
+	"github.com/jackc/pgx/v4"
 
 	"github.com/FerretDB/FerretDB/internal/handlers/common"
+	"github.com/FerretDB/FerretDB/internal/handlers/commonerrors"
+	"github.com/FerretDB/FerretDB/internal/handlers/pg/pgdb"
 	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/iterator"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
 	"github.com/FerretDB/FerretDB/internal/util/must"
 	"github.com/FerretDB/FerretDB/internal/wire"
 )
 
 // MsgCreateIndexes implements HandlerInterface.
 func (h *Handler) MsgCreateIndexes(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
-	// TODO https://github.com/FerretDB/FerretDB/issues/78
+	dbPool, err := h.DBPool(ctx)
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
 
 	document, err := msg.Document()
 	if err != nil {
-		return nil, err
+		return nil, lazyerrors.Error(err)
 	}
 
 	common.Ignored(document, h.L, "writeConcern", "commitQuorum", "comment")
 
+	command := document.Command()
+
+	db, err := common.GetRequiredParam[string](document, "$db")
+	if err != nil {
+		return nil, err
+	}
+
+	collection, err := common.GetRequiredParam[string](document, command)
+	if err != nil {
+		return nil, err
+	}
+
+	idxArr, err := common.GetRequiredParam[*types.Array](document, "indexes")
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I'm not sure in which cases we can have different errors and in which not, so just saying that this one results in different error and error code after running: `db.runCommand({createIndexes: "values", indexes: "string"})`

```
Body diff:
--- res body
+++ proxy body
@@ -15,5 +15,5 @@
         },
-        "errmsg": "required parameter \"indexes\" has type string (expected *types.Array)",
-        "code": 2,
-        "codeName": "BadValue"
+        "errmsg": "BSON field 'createIndexes.indexes' is the wrong type 'string', expected type 'array'",
+        "code": 14,
+        "codeName": "TypeMismatch"
       },
```

Reply:
I believe it will be a part of #2311 right?

Reply:
@noisersup yes, correct, it's for #2311. For the current PR, we just want to return some error.